src/DynamoCoreWpf/Views/Core/WorkspaceView.xaml

```diff
@@ -764,6 +765,11 @@
 
                     <Separator />
 
+                    <!--  Switch Python  -->
+                    <controls:ParentMenuItem  x:Name="PythonEngineMenu" Header="{x:Static p:Resources.UpdateAllPythonEngineMainMenuHeader}" Visibility="{Binding Path=CanUpdatePythonEngine, Converter={StaticResource BooleanToVisibilityCollapsedConverter }}">
```

Comment:
What is this used for?

Reply:
Yes, if no python node, or a custom node that has python nodes is selected, the option will be hidden.

Reply:
Thank you, looks good